_posts/blog/2019-02-15/2019-02-15-how-to-market-your-store-and-listings-on-openbazaar.markdown

```diff
@@ -0,0 +1,50 @@
+
+That solved the simplest part of the problem: that sellers wanted to share their links easily and potential buyers—who may or may not know about OpenBazaar yet—wanted to view them in a predictable way. 
+
+We launched [OpenBazaar.com](https://openbazaar.com) in the middle of last year and it’s been a hit. Traffic to the site is growing **around 40% month over month**. This is clearly showing us that users are interested in decentralized applications that structurally uphold some core values, but that also are accessible wherever they may be in their technological adventures.
+
+#### Sharing store & listings from within OpenBazaar
+
+People who want to share stores and listings easily on the web can easily grab their links from OpenBazaar.com several ways:
+
+1. By clicking the green “View on Web” link in the address bar within their OpenBazaar client:
+
+![OpenBazaar View on Web Link](openbazaar_view_on_web_link.png "OpenBazaar View on Web Link")
+
+2. By copying the store ID and pasting it on the end of https://openbazaar.com/store/ like this:
+
+![OpenBazaar PeerID Selected](openbazaar_peerid_selected.png "OpenBazaar PeerID Selected")
+ 
+[https://openbazaar.com/store/QmcUDmZK8PsPYWw5FRHKNZFjszm2K6e68BQSTpnJYUsML7](https://openbazaar.com/store/QmcUDmZK8PsPYWw5FRHKNZFjszm2K6e68BQSTpnJYUsML7)
+
+<br>  
+
+#### Sharing store & listings from OpenBazaar.com
+
+On OpenBazaar.com there are also additional features including these social share buttons:
+
+![OpenBazaar Widget Builder Link and Social Share Buttons](openbazaar_store_widget_builder_link.png "OpenBazaar Widget Builder Link and Social Share Buttons")
+
+There is also [the widget](https://openbazaar.com/widget) for sellers who have websites where they want to showcase their OpenBazaar store. Simply copy the code and add it to your site wherever you can post an embed or add HTML.
+
+![OpenBazaar Store Widget Builder](openbazaar_store_widget_builder.png "OpenBazaar Store Widget Builder")
+
+We are working to build a fully decentralized marketplace and social experience to promote privacy and autonomy for users around the world. It’s clear to us, though, that the road to full decentralization is made up of more decentralized steps rather than a complete jump. 
+
+Moving from centralized systems we are displeased with is a process that will take some time and the ability to make small choices through that process is imperative. Building decentralized apps that also interface with centralized apps that people are used to using right now is important to help them better match their desires for the future with their present reality.
```

Comment:
"choices" is ambiguous here. What choices are being made? Maybe "made small advances"?